content/en/opentelemetry/schema_semantics/semantic_mapping.md

```diff
@@ -20,12 +20,16 @@ OpenTelemetry makes use of a number of [semantic conventions][1] that specify na
 
 | OpenTelemetry convention | Datadog convention |
 | --- | --- |
-| `deployment.environment` | `env` |
+| `deployment.environment` <sup>1</sup>  | `env` |
```

Comment:
Does one take precedent over the other if both are set (e.g. `deployment.environment.name` takes precedence over `deployment.environment`) ? If so, we may want to mention this.

Reply:
That should never happen, one of `deployment.environment.name` or `deployment.environment` can be in one semconv version but not both

Reply:
Ah that makes sense, thanks